xml/System.Web/SameSiteMode.xml

```diff
@@ -14,7 +14,20 @@
   </Base>
   <Docs>
     <summary>Specifies constants that indicate the value for the SameSite attribute of the cookie.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks
+
+The <see cref="F:System.Web.SameSiteMode.None" /> value now causes ASP.NET to emit a SameSite cookie header to accommodate upcoming changes to SameSite cookie handling in Chrome. As part of this change, FormsAuth and SessionState cookies will be issued with SameSite = 'Lax' instead of the previous default of 'None', though these values can be overridden in web.config. 
```

Comment:
```suggestion
The <see cref="F:System.Web.SameSiteMode.None" /> value now causes ASP.NET to emit a SameSite cookie header in compliance with https://tools.ietf.org/html/draft-west-cookie-incrementalism-00 that browsers are in the process of adopting. As part of this change, FormsAuth and SessionState cookies will be issued with SameSite = 'Lax' instead of the previous default of 'None', though these values can be overridden in web.config. 
```